pandas/core/frame.py

```diff
@@ -4818,66 +4818,82 @@ def aggregate(self, func, axis=0, *args, **kwargs):
+              instead.
+            If you are just applying a NumPy reduction function this will
+            achieve much better performance.
         reduce : boolean or None, default None
-            Try to apply reduction procedures. If the DataFrame is empty,
-            apply will use reduce to determine whether the result should be a
-            Series or a DataFrame. If reduce is None (the default), apply's
-            return value will be guessed by calling func an empty Series (note:
-            while guessing, exceptions raised by func will be ignored). If
-            reduce is True a Series will always be returned, and if False a
-            DataFrame will always be returned.
-
-            .. deprecated:: 0.23.0
+            Try to apply reduction procedures. If the `DataFrame` is empty,
+            :meth:`apply` will use reduce to determine whether the result
+            should be a `Series` or a `DataFrame`. If reduce is None (the
+            default), :meth:`apply`'s return value will be guessed by calling
+            func on an empty `Series`
+            (note: while guessing, exceptions raised by `func` will be
+            ignored).
+            If reduce is True a `Series` will always be returned, and if
+            `False` a `DataFrame` will always be returned.
+
+            .. deprecated:: 0.23.0.
                This argument will be removed in a future version, replaced
                by result_type='reduce'.
 
-        result_type : {'expand', 'reduce', 'broadcast, None}
-            These only act when axis=1 {columns}:
+        result_type : {'expand', 'reduce', 'broadcast', None}
+            These only act when `axis=1` (columns):
 
             * 'expand' : list-like results will be turned into columns.
-            * 'reduce' : return a Series if possible rather than expanding
-              list-like results. This is the opposite to 'expand'.
+            * 'reduce' : returns a `Series` if possible rather than expanding
+              list-like results. This is the opposite of 'expand'.
             * 'broadcast' : results will be broadcast to the original shape
-              of the frame, the original index & columns will be retained.
+              of the `DataFrame`, the original index and columns will be
+              retained.
 
-            The default behaviour (None) depends on the return value of the
-            applied function: list-like results will be returned as a Series
-            of those. However if the apply function returns a Series these
+            The default behaviour (`None`) depends on the return value of the
+            applied function: list-like results will be returned as a `Series`
+            of those. However if the apply function returns a `Series` these
             are expanded to columns.
 
-            .. versionadded:: 0.23.0
+            .. versionadded:: 0.23.0.
 
         args : tuple
```

Comment:
`args` is always a tuple, so we remove it. The validation script will probably complain, but the convention is to use `*args` and `**kwargs` (with the stars)

Reply:
In this case we need to keep it as `args` and the 'tuple' type description is correct. This is a special case because it actually is a *keyword* with the name `args`, not `*args` in the signature

Reply:
Ok, so I let the dot at the end.

Reply:
yes, my fault, sorry.